### PR TITLE
[Release3] Merging code to master branch for release3.

### PR DESCRIPTION
**Pull request by adityagajbhiye**
HotFix Scan issues has been fixed.

### CODE REVIEW

File: modules/sde-core/src/main/resources/flyway/V15__Alter_Tables.sql [hunk 1]
```diff
@@ -0,0 +1,93 @@
+/********************************************************************************
+ * Copyright (c) 2022 T-Systems International GmbH
+ * Copyright (c) 2022 Contributors to the CatenaX (ng) GitHub Organisation
```
Comment (Siegfriedk) [inline modules/sde-core/src/main/resources/flyway/V15__Alter_Tables.sql:3]: You are mixxing catenax and eclipse foundation here. Please fix this

File: modules/sde-core/src/test/java/org/eclipse/tractusx/sde/core/controller/SubmodelCsvControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,155 @@
+package org.eclipse.tractusx.sde.core.controller;
```
Comment (carslen) [inline modules/sde-core/src/test/java/org/eclipse/tractusx/sde/core/controller/SubmodelCsvControllerTest.java:1]: License header missing

File: modules/sde-submodules/batch/src/main/java/org/eclipse/tractusx/sde/submodels/batch/BatchExecutor.java [hunk 3]
```diff
@@ -10,7 +10,7 @@
 import org.eclipse.tractusx.sde.common.submodel.executor.create.steps.impl.JsonRecordValidate;
```
Comment (carslen) [inline modules/sde-submodules/batch/src/main/java/org/eclipse/tractusx/sde/submodels/batch/BatchExecutor.java:10]: License header missing

File: modules/sde-submodules/batch/src/main/java/org/eclipse/tractusx/sde/submodels/batch/BatchSubmodel.java [hunk 4]
```diff
@@ -31,6 +31,8 @@ public void init() {
 		submodel = loadSubmodel(input);
```
Comment (carslen) [inline modules/sde-submodules/batch/src/main/java/org/eclipse/tractusx/sde/submodels/batch/BatchSubmodel.java:31]: License header missing

File: modules/sde-submodules/batch/src/main/java/org/eclipse/tractusx/sde/submodels/batch/service/BatchService.java [hunk 5]
```diff
@@ -19,7 +19,7 @@
```
Comment (carslen) [inline modules/sde-submodules/batch/src/main/java/org/eclipse/tractusx/sde/submodels/batch/service/BatchService.java:19]: License header missing

**Review by carslen — CHANGES_REQUESTED** [review]
Please ensure that all source code filed have a license header added. I've stopped checking every file for an existing header.

File: modules/sde-core/src/test/java/org/eclipse/tractusx/sde/core/controller/SubmodelCsvControllerTest.java (hunk 2 above)
Comment (adityagajbhiye) [reply modules/sde-core/src/test/java/org/eclipse/tractusx/sde/core/controller/SubmodelCsvControllerTest.java:20]: We have updated License header.

File: modules/sde-submodules/batch/src/main/java/org/eclipse/tractusx/sde/submodels/batch/BatchExecutor.java (hunk 3 above)
Comment (adityagajbhiye) [reply modules/sde-submodules/batch/src/main/java/org/eclipse/tractusx/sde/submodels/batch/BatchExecutor.java:30]: We have updated License header.

File: modules/sde-submodules/batch/src/main/java/org/eclipse/tractusx/sde/submodels/batch/BatchSubmodel.java (hunk 4 above)
Comment (adityagajbhiye) [reply modules/sde-submodules/batch/src/main/java/org/eclipse/tractusx/sde/submodels/batch/BatchSubmodel.java:51]: We have updated License header.

File: modules/sde-submodules/batch/src/main/java/org/eclipse/tractusx/sde/submodels/batch/service/BatchService.java (hunk 5 above)
Comment (adityagajbhiye) [reply modules/sde-submodules/batch/src/main/java/org/eclipse/tractusx/sde/submodels/batch/service/BatchService.java:39]: We have updated License header.

**Review by carslen — APPROVED** [review]
LGTM, wait for @SiegfriedK OK

**Pull request merged**